roles/nova-common/templates/etc/nova/policy.json

```diff
@@ -451,7 +451,7 @@
     "os_compute_api:os-server-usage:discoverable": "@",
     "os_compute_api:os-server-groups": "rule:admin_or_owner",
     "os_compute_api:os-server-groups:discoverable": "@",
-    "os_compute_api:os-services": "rule:admin_api",
+    "os_compute_api:os-services": "rule:admin_api or role:cloud_admin",
```

Comment:
Without this change, they hypervisors panel displays errors when logging in as cloud_admin